src/tests/fix-paths-dry-run/in/.env

```diff
@@ -0,0 +1 @@
+KBC_DRY_RUN=true
```

Comment:
Pridal som to do jedneho existujuceho integracneho testu, ake verifikaciu, ze to fakt funguje.